resources/repository.rb

```diff
@@ -49,7 +46,9 @@
 
     apt_repository 'rundeck' do
       uri new_resource.package_uri
-      distribution '/'
+      deb_src new_resource.debsrc
+      components %w(any main)
+      distribution ''
```

Comment:
Is this needed or can it just be removed?

Reply:
This is necessary.